src/main/java/org/osc/controller/nsfc/api/NeutronSfcSdnRedirectionApi.java

```diff
@@ -293,6 +294,17 @@ public Element registerInspectionPort(InspectionPortElement inspectionPort) thro
             throw new IllegalArgumentException("Attempt to register null InspectionPort");
         }
 
+        String inspectionPortGroupId = inspectionPort.getParentId();
+        if (inspectionPortGroupId != null) {
```

Comment:
good point. I think when the final sfc plugin can handle it however neutron handles it. 

In this case, since we do not have a port group entity in the DB, this seems like the only way to handle it. We could add port group entity in the DB but seems like too much overhead for something we expect to get rewritten without the DB anyway.

If we encounter this error in OSC, we could have the last of the DAI delete tasks clear out the parent id entry in the DS.

Reply:
Confused. I thought we were going to simply generate a new portGroupId and return that in the element's parentId instead of an exception. 

With this code, what happens if I create a DS with some ports, then delete them all, then try to register a new one under the same ds? Are we not guaranteed to get this exception?